templates/kops/default.yaml

```diff
@@ -0,0 +1,126 @@
+apiVersion: kops/v1alpha2
```

Comment:
Let's add `kops` manifest as a separate PR. I need `gomplate` to do what I'm working on.